master/static/srv/agent_setup_script.sh.template

```diff
@@ -40,6 +45,7 @@ cat /usr/local/determined/container_startup_script
 echo "#### PRINTING CONTAINER STARTUP SCRIPT END ####"
 
 docker run --init --name determined-agent {{.LogOptions}} \
+    --privileged \
```

Comment:
IIRC one particular [EE customer](https://determined-ai.slack.com/archives/C0124B04SGM/p1651260879001769?thread_ts=1651252778.066919&cid=C0124B04SGM) had their custom startup script, so when we change it on our end, it may break for them. We need to raise this when they upgrade.
cc @varlogtim